### PR TITLE
update to 1.0.0 artifacts

### PR DESCRIPTION
**Pull request by rkuhn**
see discussion at reactive-streams/reactive-streams-jvm#223

### CODE REVIEW

**Comment by benjchristensen** [comment]
+1

**Comment by viktorklang** [comment]
:+1:

**Comment by viktorklang** [comment]
Added the announcement document, it is still waiting for final confirmation regarding a couple of artifact versions. @reactive-streams/contributors Please have a quick look at https://github.com/reactive-streams/reactive-streams.github.io/commit/8f7562266d720347cb3576b46dd8e5f410581b04 ASAP

File: _includes/announce-1.0.0.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+* [Ratpack](http://www.ratpack.io) *(version `0.9.16`)*
+   * See the [“Streams”](http://www.ratpack.io/manual/current/streams.html) chapter of the manual.
+* Reactive Rabbit *(version `UNKNOWN`)*
+   * Driver for RabbitMQ/AMQP, see [here](https://github.com/ScalaConsultants/reactive-rabbit).
+* [Reactor](http://projectreactor.io/) *(version `2.0.1.RELEASE`)*
+   * For the documentation see [here](http://projectreactor.io/docs/reference/streams.html).
+* [RxJava](http://reactivex.io/) *(version `1.0.0`)*
+   * See [github.com/ReactiveX/RxJavaReactiveStreams](https://github.com/ReactiveX/RxJavaReactiveStreams).
+* [Slick](http://slick.typesafe.com/) *(version `3.0.0`)*
+   * See the [“Streaming”](http://slick.typesafe.com/doc/3.0.0-RC3/dbio.html#streaming) section of the manual.
```
Comment (szeiger) [inline _includes/announce-1.0.0.md]: The link goes to the old manual. This should be http://slick.typesafe.com/doc/3.0.0/dbio.html#streaming

Comment (viktorklang) [inline _includes/announce-1.0.0.md]: Thanks Stefan, fixing!

**Pull request merged**